workflow/rules/osemosys.smk

```diff
@@ -21,12 +21,13 @@ rule create_model_data:
     log: "results/log/copy_csv_{scenario}_{model_run}.log"
     params:
         folder=directory("results/{scenario}/model_{model_run}/data"),
+        start_year=config['start_year'],
     conda: "../envs/otoole.yaml"
     group: "gen_lp"
     output:
         csvs = expand("results/{{scenario}}/model_{{model_run}}/data/{x}.csv", x=INPUT_FILES)
     shell:
-        "python workflow/scripts/create_modelrun.py {input.csv} {params.folder} {input.sample} {input.config}"
+        "python workflow/scripts/create_modelrun.py {input.csv} {params.folder} {input.sample} {input.config} {params.start_year}"
```

Comment:
I get an error in create_modelrun.py as it the start_year is not the right type e.g. it is passed in as a string and not an int.

Reply:
I think this should now be solved by the wrapping of sys.argv[5] with int(sys.argv[5])